apisix/core/config_etcd.lua

```diff
@@ -529,6 +531,17 @@ local function _automatic_fetch(premature, self)
         return
     end
 
+    if not health_check.conf then
```

Comment:
In lua-resty-etcd
https://github.com/api7/lua-resty-etcd/blob/1fe01f77dda252f2d95eed0a7635f3479097b8ed/lib/resty/etcd/health_check.lua#L70-L83

Reply:
Is it because fail_timeout takes effect?
https://github.com/api7/lua-resty-etcd/blob/1fe01f77dda252f2d95eed0a7635f3479097b8ed/lib/resty/etcd/health_check.lua#L34

Reply:
Ah it is. Thank you so much!